packages/mds-db/migration.ts

```diff
@@ -189,12 +190,49 @@ async function recreateProviderTables(client: MDSPostgresClient) {
     }
   }
 }
+
+async function addIdentityColumnToAllTables(client: MDSPostgresClient) {
+  const exec = SqlExecuter(client)
+
+  const { rows: existing }: { rows: { table_name: string }[] } = await exec(
+    `SELECT table_name FROM information_schema.columns WHERE column_name = '${schema.IDENTITY_COLUMN}' AND table_catalog = CURRENT_CATALOG AND table_schema= CURRENT_SCHEMA`
```

Comment:
super tiny nit -- there should be a space after `table_schema`

Reply:
Fixed in multiple places